index.html

```diff
@@ -1,40 +1,98 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>The Boss</title>
-</head>
-<style>
-  .boss {
-    width: 518px;
-    height: 430px;
-    background-image: url("theboss.jpg");
-  }
-  .boss:hover {
-    background-position-y: 435px;
-  }
-</style>
-<body>
-  <div class="boss">&nbsp;</div>
-
-  <script>
-    function handleTouchStart(e) {
-      e.target.style.backgroundPositionY = "435px";
+<html lang="eng">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Boss</title>
+  </head>
+  <style>
+    *{
+      padding: 0;
+      margin: 0;
+      box-sizing: border-box;
     }
+    .boss {
+      width: 518px;
+      height: 430px;
+      background-image: url("./theboss.jpg");
+      margin: auto;
+      margin-top: 100px;
+      border: 3px solid black;
+      box-shadow: 10px;
+      border-radius: 10px;
+    }
+    .boss:hover {
+      background-position-y: 435px;
+    }
+    .head1{
+      text-align: center;
+      background-color:aqua;
+      border: 1px solid black;
+      font-weight:200 ;
+      font-size: 20px;
+      padding: 10px;
 
-    function handleTouchEnd(e) {
-      e.target.style.backgroundPositionY = "0px";
+    }
+    .span1{
+      text-align: center;
+      margin: auto;
+      background-color: #ccc;
+      border: 2px solid black;
+      width: 30%;
+      padding: 15px;
+      font-size: 24px;
+      font-weight: 600;
+      border-radius: 20px;
     }
 
-    function init() {
-      var brad = document.querySelector(".boss");
-      brad.addEventListener("touchstart", handleTouchStart, false);
-      brad.addEventListener("touchend", handleTouchEnd, false);
+    @media only screen and (max-width:992px){
+      .boss{
+        width: 100%;
```

Comment:
you can remove width... OR change the max-width in there:

 `@media only screen and (max-width:518px){`